scripts/table-regions.py

```diff
@@ -13,17 +13,28 @@ def main():
     region_info = json.loads(webContent)
 
     table_region_leader = parse_regions(region_info["record_regions"])
```

Comment:
It's better to change this function name and add `s` for all leader related variables.